components/card.html

```diff
@@ -72,26 +90,41 @@ <h2>Card with Text overlay</h2>
         <p>
           A default image is displayed for avatars when a chosen source is unavailable (for example, when there is a problem displaying the image due to an error), or simply unspecified.
         </p>
-        
-        <div class="card">
-          <div class="child-card">
-            <div class="transparent">
-              <img src="../components/images/chair.png" class="" />
-              <div class="badge-on-card">
-                #1 Bestseller
+
+        <div class="code-border">
+          <div class="code-bg">
+            <div class="card">
+              <div class="child-card">
+                <div class="transparent">
+                  <img src="../components/images/chair.png" class="" />
```

Comment:
Suggestion:- This is no alt attribute please add alt attribute. The class attribute is also empty.